Scripts/RemediationScripts/Remediate-ConfigureIoTHubTLSVersion.ps1

```diff
@@ -365,7 +365,10 @@ function Set-MinTLSVersionForIoTHub  {
 
     $IoTHubDetails | ForEach-Object {
         $IoTHub = $_
-        if(!$_.MinimumTlsVersion -or $_.MinimumTlsVersion -lt $requiredMinTLSVersion)
+        $ApplicableRegions = @("eastus", "southcentralus", "westus2")
+        $IsRegionSupportedforTLSEnforcement = $ApplicableRegions -contains $_.Location
+
+        if((!$_.MinimumTlsVersion -or $_.MinimumTlsVersion -lt $requiredMinTLSVersion) -and $IsRegionSupportedforTLSEnforcement)
```

Comment:
Please make sure to update zip post approval on these changes

Reply:
Pushed changes updating the zip file with latest changes